packages/portal-proto/src/components/InputEntityList/InputEntityList.unit.test.tsx

```diff
@@ -19,47 +19,47 @@ jest.spyOn(core, "fetchGdcEntities").mockResolvedValue({
   },
 } as any);
 
+const createSet = jest.fn();
+const createSetHook = jest.fn().mockReturnValue([createSet, {}]);
+
+const themeColors = Object.fromEntries(
```

Comment:
Can you move this to the render wrapper function in `test-utils` and then use that instead?

Reply:
Done!